pyproject.toml

```diff
@@ -26,7 +34,6 @@ exclude = [
 ]
 packages = [

```

Comment:
I've been doing a bit of reading on how poetry likes to see things set up, and as it turns out `docs/` and `tests/` should not be in this attribute, but rather just be made part of the `include` attribute :).